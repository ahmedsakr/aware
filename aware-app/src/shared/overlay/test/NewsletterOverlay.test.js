import React from 'react';
import { shallow } from 'enzyme';

import NewsletterOverlay from './NewsletterOverlay';

describe('NewsletterOverlay', () => {
  it('should render correctly in "debug" mode', () => {

    const component = shallow(<NewsletterOverlay debug />);
  
    expect(component).toMatchSnapshot();
  });
});