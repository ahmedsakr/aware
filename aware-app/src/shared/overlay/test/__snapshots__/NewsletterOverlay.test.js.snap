// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsletterOverlay should render correctly in "debug" mode 1`] = `
<AwareOverlay
  content={
    <div
      class="content-layer"
    >
      <div
        id="subscribe-blurb"
      >
        Once you are subscribed to the Aware newsletter, you will receive weekly e-mails on the latest development updates we are pushing to Aware. We are actively working on enhancing the product that you love.
      </div>
      <div
        id="developer-pictures"
      >
        <div
          class="developer"
        >
          <p>
            Ahmed Sakr
          </p>
          <img
            alt="ahmed"
            src="/ahmed-pic.jpg"
          />
        </div>
        <div
          class="developer"
        >
          <p>
            Josh Campitelli
          </p>
          <img
            alt="josh"
            src="/josh-pic.jpg"
          />
        </div>
      </div>
      <div
        id="subscribe-data"
      >
        <p>
          You can unsubscribe at any time!
        </p>
        <input
          class="form-control"
          placeholder="john@example.com"
          type="email"
        />
      </div>
    </div>
  }
  footer={
    <div
      id="footer-layer"
    >
      <button
        class="btn btn-secondary"
        data-dismiss="modal"
        type="button"
      >
        I've changed my mind
      </button>
      <button
        class="btn btn-primary"
        type="button"
      >
        Subscribe
      </button>
    </div>
  }
  name="newsletter-overlay"
  title="Subscribe to our Newsletter"
/>
`;
